internal/servers/controller/controller.go

```diff
@@ -146,6 +158,9 @@ func (c *Controller) Start() error {
 		return nil
 	}
 	c.baseContext, c.baseCancel = context.WithCancel(context.Background())
+	if err := c.scheduler.Start(c.baseContext); err != nil {
```

Comment:
Since the scheduler will currently not run any jobs (non exist). Should I replace this with a `TODO` so we don't have the goroutine running for now?

Reply:
I think it's fine as-is.